frontend/ui/widget/touchmenu.lua

```diff
@@ -594,16 +594,26 @@ function TouchMenu:updateItems()
     table.insert(self.item_group, self.footer_top_margin)
     table.insert(self.item_group, self.footer)
     -- @translators %1 is the current page. %2 is the total number of pages. In some languages a good translation might need to reverse this order, for instance: "Total %2, page %1".
```

Comment:
The comment should remain attached to the string for clarity. I think xgettext will simply link it to the first following string.